Apps/AdminWebClient/src/ClientApp/src/components/core/modals/BannerModal.vue

```diff
@@ -25,8 +25,7 @@
                                 "
                             >
                                 <v-datetime-picker
-                                    v-model="editedItem.effectiveDateTime"
-                                    requried
```

Comment:
is the required attribute not needed?

Reply:
It doesn't actually do anything because vuetify-datetime-picker doesn't support it.  I created a bug to address that (AB#10889).